server/routes/bookVisit/mainContactController.test.ts

```diff
@@ -19,8 +19,14 @@ let sessionData: SessionData
 const bookerReference = TestData.bookerReference().value
 const prisoner = TestData.prisoner()
 const prison = TestData.prisonDto()
-const adultVisitor = TestData.visitor()
-const childVisitor = TestData.visitor({ dateOfBirth: `${new Date().getFullYear() - 2}-01-01`, adult: false })
+const adultVisitor1 = TestData.visitor({ visitorDisplayId: 1, visitorId: 100 })
```

Comment:
Making sure the test contacts have different IDs (otherwise they are all defaulting to `1`). Add another contact so it can be confirmed that main contact choice is only from `selectedVisitors`, which is a subset of `allVisitors`.